src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/OrderlyServerAPI.kt

```diff
@@ -89,10 +89,25 @@ class OrderlyServer(
 
     override fun get(url: String, context: ActionContext): OrderlyServerResponse
     {
-        val request = Request.Builder()
-            .url(buildFullUrl(url, context.queryString()))
-            .headers(standardHeaders.toHeaders())
-            .build()
+        val request: Request
+        when
+        {
+            (context.queryParams("report-name").isNullOrEmpty()) ->
```

Comment:
I can see why this is necessary (because otherwise the unmodifiable context "pollutes" the next call) but we should find a way to make it unnecessary. See comment on `runReportInline.vue` below.